extensions/amp-ngage-livechat/0.1/amp-ngage-livechat.css

```diff
@@ -0,0 +1,255 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+.ngageMobileBar {
+  background-color: rgb(240, 140, 2);
+}
+a.ngageBoxClose {
+  float: right;
+  margin-top: 4px;
+  margin-right: 4px;
+  cursor: pointer;
+  color: #ccc;
+  background: 0 0;
+  font-size: 48px;
+  display: inline-block;
+  line-height: 24px;
+  font-family: 'Roboto'
+}
+.ngageStrong {
+  font-weight: 700;
+  display: inline-block
+}
+.ngageBoxClose:before {
+  content: '×'
+}
+.ngageBoxClose:after {
+  content: '';
+  height: 2px;
+  background: #ccc;
+  display: block;
+  margin-top: 2px
+}
+.ngageHeader {
+  font-family: 'Roboto Slab';
+  font-size: 125%
+}
+.ngageMobileFloat {
+  z-index: 99999999;
+  position: fixed;
+  transition: top .5s;
+  top: 0;
+  left: 0;
+  width: 100%;
+  background-color: #333;
+  color: #fff;
+  font-family: 'Arial';
+  font-size: 16px;
+  box-shadow: 0 20px 10px #888
+}
+.ngageText {
+  padding: .5em 5% 2em
+}
+.ngageMobilePicContainer {
+  margin-right: 5%;
+  margin-top: 8px;
+  float: left;
+  height: 60px;
+  width: 60px;
+  overflow: hidden;
+  background-color: #fff;
+  display: inline-block
+}
+#ngageMobilePic {
+  margin: 0 auto;
+  height: 100%;
+  width: auto
+}
+.ngageMobileBar {
+  position: absolute;
+  bottom: -1em;
+  padding: .25em 1em;
+  text-align: center;
+  font-size: 16px;
+  color: #fff;
+  width: 60%;
+  border-radius: 1em;
+  margin-left: 20%
+}
+.ngageMobileText {
+  display: inline;
+  margin-left: 5px;
+  font-size: 22px
+}
+.liveChatFloatingButtonMobile {
+  transition: bottom .5s
+}
+.ngageMobilePicContainer {
+  border-radius: 0;
+  border: 0
+}
+@keyframes dropHeader {
+  0%, 89% {
+    transform: translateY(-300px)
+  }
+  to {
+    transform: translateY(0)
+  }
+}
+@keyframes raiseHeader {
+  0% {
+    transform: translateY(0)
+  }
+  to {
+    transform: translateY(-300px)
+  }
+}
+#ngageMobileInvite {
+  display: none
+}
+.ngageMobileFloat {
+  animation-name: dropHeader;
+  animation-fill-mode: forwards;
+  animation-iteration-count: 1;
+  animation-timing-function: ease-out;
+  animation-duration: 5.6s
+}
+#ngageMobileInvite:checked+label {
+  animation-name: raiseHeader;
+  animation-iteration-count: 1;
+  animation-timing-function: ease-out;
+  animation-duration: .6s
+}
+.liveChatFloatingButtonMobile {
+  background-color: #474747;
+  border-radius: 0;
+  text-indent: 0;
+  border: 1px solid #dcdcdc;
+  color: #e8e8e8;
+  box-shadow: 5px -2px 5px #888;
+  font-family: 'Roboto Slab Regular', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-size: 27px;
+  font-weight: 700;
+  font-style: normal;
+  height: 52px;
+  line-height: 52px;
+  width: 100%;
+  display: block;
+  bottom: 0;
+  padding: 0;
+  position: fixed;
+  z-index: 5000;
+  text-decoration: none;
+  text-align: center;
+  text-rendering: optimizeLegibility
+}
+@font-face {
```

Comment:
Font faces must use [`<amp-font>`](https://github.com/ampproject/amphtml/blob/master/extensions/amp-font/amp-font.md#-amp-font) to avoid browser's poor preloading.